.github/workflows/wss-scan.yml

```diff
@@ -0,0 +1,46 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 3 * * *'
+  repository_dispatch:
+    types: [wss-scan]
```

Comment:
Considering using a matching pattern to existing workflows in naming convention.
For example: check-vulnerability-whitesource
than a very vague and unmatched name like wss-scan, which no one knows what it is on the first place just by reading it